### ISSUE TITLE
Brotli support

### ISSUE THREAD

**Issue by dominikkrulak** [bitnami/bitnami-docker-nginx#131]
Are you planning to support the Brotli module?
Thank you

**Comment by javsalgar** [comment]
Hi,

We did not have immediate plans to support it but it's something we can consider. Could you explain your use case?

**Comment by DomKrulak** [comment]
Hi there,

I'll sketch down some top points why I'd like you to add the Brotli support:

- smaller files like JS, CSS by significant size difference against Gzip
- better compression ratio, compression speed, and decompression speed (Google some tests)
- less bandwidth used therefore a faster page load
- if using Content Delivery Network (CDN), less data is transferred to CDN
- supported by major browsers except IE (if browser doesn't support Brotli, Gzip is used instead)
- Google maintained

To sum it up it's about the file size and page speed.

Thank you for your consideration.

**Comment by javsalgar** [comment]
Thank you very much for your input. I will create an internal task with the information you provided so it can be evaluated. If we move forward with this, I will update this ticket.

**Comment by galaczi** [comment]
Any chance of this happening?

**Comment by javsalgar** [comment]
Hi,

I'm afraid it is still in our backlog. I will update this ticket when we have more information about this. We have also requests for adding the stream module as well, so I will bring up the subject again to see if we can reconsider the priority.

**Comment by javsalgar** [comment]
Hi,

Just a quick note to let you know that we released a new version of the container with brotli

```
I have no name!@79ea05e9e5c8:/app$ nginx -V
nginx version: nginx/1.18.0
built by gcc 8.3.0 (Debian 8.3.0-6)
built with OpenSSL 1.1.1d  10 Sep 2019
TLS SNI support enabled
configure arguments: --prefix=/opt/bitnami/nginx --with-http_stub_status_module --with-stream --with-http_gzip_static_module --with-mail --with-http_realip_module --with-http_v2_module --with-http_ssl_module --with-mail_ssl_module --with-http_gunzip_module --with-http_auth_request_module --with-http_sub_module --with-http_geoip_module --add-module=/bitnami/blacksmith-sandox/nginx-module-vts-0.1.18 --add-dynamic-module=/bitnami/blacksmith-sandox/nginx-module-geoip2-3.3.0 --add-module=/bitnami/blacksmith-sandox/nginx-module-substitutions-filter-0.20190806.0 --add-module=/bitnami/blacksmith-sandox/nginx-module-brotli-0.20200423.0
```

Do not hesitate to re-open the ticket if you find any issue
